api/users/serializers.py

```diff
@@ -104,6 +104,17 @@ def update(self, instance, validated_data):
             instance.save()
         except ValidationValueError as e:
             raise InvalidModelValueError(detail=e.message)
+        # ValidationError s raised by instance.save() are likely to have e.args[0] be
+        # dict of the form {'field validated': [<Exception>]}
+        except ValidationError as e:
+            if isinstance(e.args[0], dict):
+                value = e.args[0].popitem()[1]
+                if len(value) == 1:
```

Comment:
Per our conversation yesterday EOB. After further review, I learned that these exceptions are handled by a custom handler that takes all these possibilities into consideration. I'm not thrilled with the formating of the message it produces, but the pertinent information is available to API user.
  Therefore, I have updated to simply re-raise the correct Exception.